services/director-v2/tests/integration/02/utils.py

```diff
@@ -230,7 +230,7 @@ async def assert_retrieve_service(
         f"/v2/dynamic_services/{service_uuid}:retrieve",
         json=dict(port_keys=[]),
         headers=headers,
-        allow_redirects=True,
+        follow_redirects=True,
```

Comment:
is this an options renamed in the new a API or a fix?

Reply:
the API changed here, had to adapt the code as well.